### DISCUSSION TITLE
Broken image on providers page

### DISCUSSION THREAD

**Discussion by kf106** [nextauthjs/next-auth#2914] [Help]
"next-auth": "4.0.0-beta.4": The page that shows the list of providers has a broken image:

![image](https://user-images.githubusercontent.com/25053735/136352736-92a1cb87-9c4c-4e48-9ef8-7c714f70e01d.png)

I'm pretty sure it needs to be specified somewhere in `/pages/api/[...nextauth].js` but I can't find anything in the docs that says how the providers page image is set.

Can someone point me in the right direction?

**Comment by kf106** [comment]
Actually, it looks like this might be a bug, as I haven't defined a theme.logo in styles, and so it's setting it as src(unknown).

**Comment by balazsorban44** [comment]
Thanks, probably introduced in #2788, will fix this soon. We should have rendered the image conditionally, when the user did not define a brand image

**Comment by kf106** [comment]
Update: worked it out. For some reason I thought it was CSS, but it just goes in the [...nextauth].js file, and make sure there's a comma after the last } if you don't put it at the end.

Where does the following snippet go? in the [...nextauth].js file, or the _app.js file?
```
theme: {
  colorScheme: "auto", // "auto" | "dark" | "light"
  brandColor: "", // Hex color code
  logo: "" // Absolute URL to image
}
```

**Reply by balazsorban44** [reply]
you won't have to do anything, @ndom91 will release a fix. we simply have to conditionally render the brand logo, if you don't define one.

**Reply by balazsorban44** [reply]
Fix on the way in #2916

**Reply by ndom91** [reply]
PR opened. Gonna collect them hacktoberfest PRs :money_mouth_face:  :joy:
